src/processor/operator/ddl/create_table.cpp

```diff
@@ -10,6 +11,10 @@ namespace kuzu {
 namespace processor {
 
 void CreateTable::executeDDLInternal(ExecutionContext* context) {
+    for (auto& child : children) {
```

Comment:
I'm against this change. In this design, `CreateTable` is no longer source operator so it does not match our definition of `SimpleOperator`. Also the right way to call a child operator is to use `getNextTuple()`. This code also make a strong assumption that if `CreateTable` has a child, then it must be `CreateSequence` which is not good for future maintenance .

So I would just do `catalog->createSequence();` here instead of calling child operator. Let me know if this is not doable.

Reply:
I agree. Going to revert these changes. I think I will handle it in the catalog instead to resolve any nesting (i.e. relTableGroup)